.github/workflows/build.yml

```diff
@@ -18,20 +22,23 @@ jobs:
         with:
           fetch-depth: 0  # fetches all commits/tags
 
-      - name: Set dynamic variables
-        # this might seem odd but is the recommended way to set variables
-        # reference: https://docs.github.com/en/actions/using-workflows/workflow-commands-for-github-actions#setting-an-environment-variable
-        run: |
-          pip install poetry
-          BUILD_VERSION=$(poetry version -s)
-          echo "BUILD_VERSION=$BUILD_VERSION+$GITHUB_SHA" >> $GITHUB_ENV
-          echo "IMAGE_NAME_WITH_TAG=quipucords:$BUILD_VERSION" >> $GITHUB_ENV
-
-      - name: Docker Layer Caching
-        uses: satackey/action-docker-layer-caching@v0.0.11
-        # Ignore the failure of a step and avoid terminating the job.
-        continue-on-error: true
-
       - name: Build quipucords image
-        run: docker build --build-arg BUILD_COMMIT=$GITHUB_SHA . -t $IMAGE_NAME_WITH_TAG
+        id: build-image
+        uses: redhat-actions/buildah-build@v2
+        with:
+          image: quipucords/quipucords
```

Comment:
It's easy to also build arm image: just install qemu-user-static and add `platforms` here. See https://github.com/mirekdlugosz/quipucords/commit/b2aaf991a8fa5a2a593252fbeaa5a61d992d5673

HOWEVER building amd64 image takes about 2 minutes. Building arm image takes 10 minutes. And we upload only once both finished building.

Jenkins pipeline will need to pull image built here, and if we spend 12 minutes building images, and then 30 minutes running Camayoc, you will get Camayoc results about 45 minutes after opening PR. I set it as a self-imposed goal to have Camayoc results within 30 minutes after opening PR.

There might be a way to build and upload amd64, and then build and upload arm separately. I have not investigated. I suggest we made that a separate task.